distr_tracing/distr_tracing_otel/distr-tracing-otel-installing.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-otel-installing"]
+= Installing distributed tracing data collection OpenTelemetry
+include::_attributes/common-attributes.adoc[]
+:context: install-distributed-tracing-otel
+
+toc::[]
+
```

Comment:
[pavolloffay](https://github.com/pavolloffay), I'm not acquainted with this product. I noticed that this page is missing the following paragraphs that are in https://61948--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-installing.html. Is this a deliberate omission? Or are these paragraphs not relevant to the content on this page?
![image](https://github.com/openshift/openshift-docs/assets/78564304/07464436-2258-4bc5-ac9e-1a8585068a07)

Reply:
Correct, and this is a mistake. The OpenTelemetry does not integrate with service mesh in any form. The service mesh docs do not contain anything about the OTEL operator.